src/NzbDrone.Core/Datastore/DbFactory.cs

```diff
@@ -72,7 +84,30 @@ public IDatabase Create(MigrationContext migrationContext)
                     }
             }
 
-            _migrationController.Migrate(connectionString, migrationContext);
+            try
+            {
+                _migrationController.Migrate(connectionString, migrationContext);
+            }
+            catch (SQLiteException ex)
+            {
+                var fileName = _connectionStringFactory.GetDatabasePath(connectionString);
+
+                if (migrationContext.MigrationType == MigrationType.Log)
+                {
+                    Logger.Warn(ex, "Logging database is corrupt, attempting to recreate it automatically");
```

Comment:
I went with warning because we're attempting to fix it, if it fails again we'll end up logging another error, which we can catch and throw a bit nicer.